src/components/memorygame/MemoryGameFactList/index.jsx

```diff
@@ -1,31 +1,65 @@
 import React from "react";
 import "./index.css";
 import { List, Card } from "antd";
+import { useTranslation } from "react-i18next";
 
 /**
  * Upon matching cards, a fact related to matched cards appear on screen
  * This component creates the list that will hold all facts
  * "facts" is an array of objects
- * "title" is a string that is translated by i18n. It is used to describe what the list holds.
  * If the language is English "title" will be "Facts"
+ * "emptyFactListMessage" is a string that is translated by i18n.
+ * It is displayed when fact list is empty
+ * If the language is English "emptyFactListMessage" will be ""Match cards to reveal facts!"
  */
-const MemoryGameFactList = ({ facts, title }) => {
-  return (
-    <>
-      <h1 className="memoryGame__title">{title}</h1>
+const MemoryGameFactList = ({ facts }) => {
+  // facts example:
+  // {
+  //   cardKey: "alternateEnergy0",
```

Comment:
again i dont recommend doing soemthing like this in futrue projects but its fine for now. generally you'd want to use something like typescript to define an interface. the problem with this approach is that its very brittle to code changes, there is nothing stopping you from adding/removnig a field from facts and forgetting that this comment exists.

Reply:
PTAL